homeassistant/auth/__init__.py

```diff
@@ -175,6 +220,45 @@ def auth_providers(self) -> List[AuthProvider]:
 
         await self._store.async_remove_credentials(credentials)
 
+    async def async_enable_user_mfa(self, user, mfa_module_id, data):
+        """Enable a multi-factor auth module for user."""
+        if mfa_module_id not in self._mfa_modules:
```

Comment:
I would expect this error to be done by checking `if module is None` a couple of lines below.